### PR TITLE
feat: add widget form sidebar to view mode modal

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
Finally!
Only markup

스크린샷 2023-07-26 오후 3 09 53

### CODE REVIEW

File: apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue [hunk 1]
```diff
@@ -464,10 +465,19 @@ export default defineComponent<Props>({
     height: 100%;
     gap: 1rem;
     overflow-y: auto;
-    .p-field-group {
-        margin: 0;
-        .input {
-            width: 100%;
+
+    /* custom design-system component - p-field-title */
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:469]:
```suggestion
/* custom design-system component - p-field-group */
    /* custom design-system component - p-field-title */
```

Comment (WANZARGEN) [reply apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue:469]: suggestion: I think two components' css is customized.

**Review by WANZARGEN — APPROVED** [review]
LGTM~

**Review by skdud4659 — APPROVED** [review]
LGTM!

**Pull request merged**